PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -217,7 +217,12 @@ def miniAOD_customizeCommon(process):
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
-    switchOnVIDPhotonIdProducer(process,DataFormat.MiniAOD) 
+    switchOnVIDPhotonIdProducer(process,DataFormat.AOD) 
```

Comment:
the logic here was the following:
- the input photon collection at this point is "reducedEgamma","reducedGedPhotons" (I think slimmedPhotons didn't work because it caused a circular dependence problem)
- "reducedEgamma","reducedGedPhotons" doesn't have associatedPackedPFCandidates() available because it is not a pat::Photon
- the way CITK is written is either use associatedPackedPFCandidates() for footprint and packedPFCanidates or use particleBasedIsolation map and particleFlow collection (and derived pfNoPileUpCandidates). Basically, the only thing depends in switchOnVIDPhotonIdProducer() on the dataFormat is the isolation part. So by using AOD here, it means that particleBasedIsolation map and pfNoPileUpCandidates should be used.